specification/baremetalinfrastructure/resource-manager/Microsoft.BareMetalInfrastructure/BareMetalInfrastructure/readme.csharp.md

```diff
@@ -11,5 +11,5 @@ csharp:
   clear-output-folder: true
   client-side-validation: false
   namespace: Microsoft.BareMetalInfrastructure
-  output-folder: $(csharp-sdks-folder)/baremetalinfrastructure/management/Microsoft.BareMetalInfrastructure/GeneratedProtocol
+  output-folder: $(csharp-sdks-folder)/baremetalinfrastructure/management/./GeneratedProtocol
```

Comment:
why do we need to change this? Could you revert it back? Thanks

Reply:
Have reverted.